demos/motion_tracking/motion_tracking_network.py

```diff
@@ -0,0 +1,229 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import typing as ty
+from multiprocessing import Pipe
+from lava.lib.dnf.connect.connect import connect
+from lava.magma.core.run_conditions import RunSteps
+from demos.motion_tracking.dvs_file_input.process import DVSFileInput, PyDVSFileInputPM
+from lava.proc.lif.process import LIF
+from lava.proc.embedded_io.spike import NxToPyAdapter, PyToNxAdapter
+from lava.lib.dnf.kernels.kernels import MultiPeakKernel, SelectiveKernel
+from lava.lib.dnf.operations.operations import Convolution, Weights
+from demos.motion_tracking.process_out.process import ProcessOut, DataRelayerPM
+from lava.magma.core.run_configs import Loihi2HwCfg
+from demos.motion_tracking.rate_reader.process import RateReader
+from lava.magma.compiler.compiler import Compiler
+from lava.magma.core.process.message_interface_enum import ActorType
+from lava.magma.runtime.runtime import Runtime
+from lava.magma.compiler.subcompilers.nc.ncproc_compiler import CompilerOptions
+
+CompilerOptions.verbose = True
+
+# Default configs for the motion tracking network
+dvs_file_input_default_config = {
+    "true_height": 180,
+    "true_width": 240,
+    "file_path": "dvs_recording.aedat4",
+    "flatten": False,
+    "downsample_factor": 8,
+    "downsample_mode": "max_pooling"
+}
+
+multipeak_dnf_default_config = {
+    "in_conn": {
+        "weight": 8
+    },
+    "lif": {"du": 2000,
+            "dv": 2000,
+            "vth": 30
+
+            },
+    "rec_conn": {
+        "amp_exc": 14,
+        "width_exc": [5, 5],
+        "amp_inh": -10,
+        "width_inh": [10, 10]
+    },
+    "out_conn": {
+        "weight": 20,
+    }
+}
+
+selective_dnf_default_config = {
+    "lif": {"du": 809,
+            "dv": 2047,
+            "vth": 30
+            },
+    "rec_conn": {
+        "amp_exc": 7,
+        "width_exc": [7, 7],
+        "global_inh": -5
+    }
+}
+
+rate_reader_default_config = {
+    "buffer_size": 10,
+}
+
+
+class MotionNetwork:
```

Comment:
Suggesting to rename to `MotionTracker`.

Reply:
Done